alf/examples/sac.gin

```diff
@@ -0,0 +1,8 @@
+import alf.algorithms.agent
```

Comment:
Not sure if it is good to use this shared sac.gin file. It does not save too many lines of shared code while making the original gin file less self-contained.

Reply:
> Not sure if it is good to use this shared sac.gin file. It does not save too many lines of shared code while making the original gin file less self-contained.

Just similar to 'ppo.gin'.